HHS/Head-Start-TTADP
# TTAHUB188/189 - Improvements for Node app to improve production performance

## Description of change

Alerted last week to an issue with a user being unable to access an activity report they'd created. This led a larger investigation of how our backend handles large activity reports, and a number of solutions were discussed. Within this pull request are two of these solutions

- Sequelize, our database ORM, was generating queries that were large enough to crash the production DB. Research indicated there was a parameter we could pass to instruct Sequelize to create several smaller queries. This reduces strain on our instance.
- Express, our API framework, has a default size for requests that it can accept. It was possible to create an activity report that when reduced to pure text, was simply too large for Express to handle. We raised the limit for Express significantly.

For exhaustive details: 
https://docs.google.com/document/d/1QgIbxHWvXEeHPxAMcv8CX4jykO5k_70N/edit#heading=h.2vohvrxjnfbd

## How to test
Users should now be able to create and edit large activity reports. In addition, engineers have verified increased capabilities and reduced queries through browser tools, New Relic, and Cloud.gov logs

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-188
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-189

## Checklists

### Every PR



**Not applicable**

### Production Deploy


### After merge/deploy
